src/components/grid-aware/PartnersAndSponsorsBlock/PartnersAndSponsorsBlock.module.css

```diff
@@ -0,0 +1,67 @@
+/* Main grid */
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
```

Comment:
You'll want to override this in a media query for smaller screen sizes, since this isn't appropriate for mobile. See https://github.com/ShelterTechSF/sheltertech.org/blob/559e1cf695ed06c1d3d6f38e14b23d4cd6e5f102/src/components/grid-aware/ThreeParagraphBlock/ThreeParagraphBlock.module.css#L48-L51.